fbw-a380x/src/wasm/fbw_a380/src/model/FacComputer_data.cpp

```diff
@@ -1906,14 +1906,14 @@ FacComputer::Parameters_FacComputer_T FacComputer::FacComputer_P{
   { 0.0, 1.0, 2.0, 3.0, 4.0, 5.0 },
   1.23,
 
-  { 230.0, 200.0, 200.0, 185.0, 177.0, 177.0 },
+  { 263.0, 220.0, 220.0, 196.0, 182.0, 182.0 },
```

Comment:
```suggestion
  { 263.0, 220.0, 220.0, 196.0, 182.0, 182.0 },
```

Are these changed in the simulink models too?

Reply:
Unfortunately no, I don't have Simulink and the files are in a binary format. Do you know someone who could update them? Should be fairly quick

Reply:
@lukecologne Could take a look at this at some point if/when he's available.